halo/app/app.go

```diff
@@ -136,9 +142,19 @@ func newApp(
 		&app.EvidenceKeeper,
 		&app.UpgradeKeeper,
 		&app.SlashingEventProc,
-		&app.StakingEventProc,
 		&app.UpgradeEventProc,
-	); err != nil {
+	}
+
+	// TODO (christian): remove feature check with logs
+	if feature.FlagEVMStakingModule.Enabled(ctx) {
+		log.Info(ctx, "EVMStakingModule feature ENABLED")
```

Comment:
We already log which feature flags are enabled in `feature.WithFlags`, so can remove these logs as they are duplicate.